docs/deployment/gunicorn.rst

```diff
@@ -0,0 +1,32 @@
+********
+gunicorn
+********
+
+The short story
+===============
+Running your pyramid based application with gunicorn can be as easy as::
+
+    gunicorn --paste production.ini
+
```

Comment:
Are there any settings in the `ini` that are specific to gunicorn? If so, it would be good to include examples here.